src/handle.ml

```diff
@@ -77,6 +77,49 @@ let handle_tactic : sig_state -> Proof.t -> p_tactic -> Proof.t =
   | P_tac_print
   | P_tac_proofterm
   | P_tac_focus(_)      -> assert false (* Handled above. *)
+  | P_tac_assert(h,t)     ->
+     begin
+       (* Obtaining the goals environment, type and meta variable. *)
+       let env = fst (Proof.Goal.get_type g) in
+       let t = fst (Scope.scope_term StrMap.empty ss env t) in
+       let m = Proof.Goal.get_meta g in
+       (* Enriching the environnement with a var of type t *)
+       let h = match h with
```

Comment:
To be consistent with the rest of the code, the `match` should be on the following line (same thing below).